pages/coordinator.vue

```diff
@@ -1,20 +1,147 @@
 <script setup>
+import { ref } from "vue";
 import ProfileInfo from "~/components/admin/ProfileInfo.vue";
 import Table from "~/components/Coordinator/Table.vue";
+import Modal from "~/components/Modal/Modal.vue";
+
+const modalVisibility = ref(false);
+const currentTeam = ref("Perth Modern");
+
+const teamState = {
+  levelPresent: false,
+  numberOfTeams: 0,
+  tuesdayAllocation: 0,
+  wednesdayAllocation: 0,
+};
+const formInput = ref({
+  tournament: null,
+  teams: [
+    { teamLevel: "Novice", ...teamState },
+    { teamLevel: "Junior", ...teamState },
+    { teamLevel: "Senior", ...teamState },
+  ],
+});
+const updateLevels = (chips) => {
+  // if level no longer selected, delete form data for corresponding level
+  formInput.value.teams = formInput.value.teams.map((team) => {
+    if (chips.includes(team.teamLevel)) {
+      return { ...team, levelPresent: true };
+    } else {
+      return { teamLevel: team.teamLevel, ...teamState };
+    }
+  });
+};
+const saveTeamRegistration = async () => {
+  // TODO:
```

Comment:
```suggestion
  // TODO:
```
IMO these belong in the GitHub issues rather than the source code.

Reply:
Leaving it in to help backend connect firebase